piper/ratbagd.py

```diff
@@ -376,7 +405,8 @@ def get_svg(self, theme):
 
         @param theme The theme from which to retrieve the SVG, as str
         """
-        return self._dbus_call("GetSvg", "s", theme)
+        fd = self._dbus_call_return_one_fd("GetSvg", "s", theme)
```

Comment:
For Piper at least, we could also do all the reading here and return a RSVG handle here. That would move all the reading to a single place, where we can reset the fd and remove the duplication now present in the device row and the mouse map. This is my preferred solution.

How would this work for the bindings for ratbagctl?

Reply:
well, we don't know in the bindings what the SVG is being used for. Even in piper we use it as both SVG and XML file (hence the need to `seek(0)`).  I have this fixed as simply calling the DBus API twice now, this way it resembles the original code (where we opened the file twice too)

Reply:
Oh, and: to avoid the duplication we'd need some util function in some module, etc. All ends up being more complicated than two lines of duplicated code, so I don't think it's worth it

Reply:
Right, I forgot about the XML use case.